bokehjs/src/less/continuum.less

```diff
@@ -16,7 +16,7 @@
 .tableheader {
   background-color: #C0C0C0;
 }
-#notebook .bk-plot-wrapper table {
```

Comment:
I'm not sure why it was added originally, but at least preliminary testing indicates that it does not seem needed (and I do think those styles should be universally applied?) and as you indicate it really needs to go in any case.